backend/local/socket.ts

```diff
@@ -19,25 +21,25 @@ export const startWebSocketServer = () => {
     const socketId = generateId(16);
     (ws as WebsocketWithId).id = socketId;
 
-    const config = {
+    const config: ConfigWithHandler = {
       connectionId: socketId,
       tableName: 'scrum-poker-local',
       ddb,
       handler: {
-        postToConnection: ({ ConnectionId, Data }: { ConnectionId: string; Data: unknown }) => {
+        postToConnection: ({ ConnectionId, Data }) => {
+          if (!Data) return;
           const client = [...wss.clients].find(
             (client) => ConnectionId === (client as WebsocketWithId).id
           );
-          const resultPromise = client
-            ? new Promise<void>((resolve, reject) =>
-                client.send(Data, (error) => (error ? reject(error) : resolve()))
+          return client
+            ? new Promise<PostToConnectionCommandOutput>((resolve, reject) =>
+                client.send(Data, (error) => (error ? reject(error) : resolve({ $metadata: {} })))
```

Comment:
What is the reason for `$metadata: {}`?

Reply:
The types require it in `PostToConnectionCommandOutput`. Otherwise, no idea.